bundles/binding/org.openhab.binding.ihc/src/main/java/org/openhab/binding/ihc/internal/IhcGenericBindingProvider.java

```diff
@@ -169,10 +276,25 @@ public int getRefreshInterval(String itemName) {
 	}
 
 	@Override
-	public boolean isOutBindingOnly(String itemName) {
+	public boolean isOutBinding(String itemName, int resourceId) {
 		IhcBindingConfig config = (IhcBindingConfig) bindingConfigs
 				.get(itemName);
-		return config != null ? config.outBindingOnly : null;
+		
+		boolean isOutBinding = true;
+		
+		if(config.resourceId == resourceId) {
+			isOutBinding = false;
+		}
+		
+		
+		return isOutBinding;
+	}
+	
+	@Override
+	public boolean hasInBinding(String itemName) {
+		IhcBindingConfig config = (IhcBindingConfig) bindingConfigs.get(itemName);
+		
+		return config != null ? config.resourceId > 0 : null;
```

Comment:
OK, now I understand :) It's sometimes difficult to follow code just in PR diff, sorry about that.